types/superagent-promise/superagent-promise-tests.ts

```diff
@@ -0,0 +1,5 @@
+// via: http://visionmedia.github.io/superagent/
+import * as superagentPromise from 'superagent-promise';
+import * as superagent from 'superagent';
+
+superagentPromise.superagent_promise(superagent, global.Promise); // $ExpectedType any
```

Comment:
This does not look like a valid use of this library. see https://github.com/lightsofapollo/superagent-promise#readme for examples. i would recommend using the sample in the readme for this.